<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SystemWindow</class>
 <widget class="QWidget" name="SystemWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>597</width>
    <height>341</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>屏保设置：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QDateTimeEdit" name="DateTimeEdit">
     <property name="buttonSymbols">
      <enum>QAbstractSpinBox::NoButtons</enum>
     </property>
     <property name="currentSection">
      <enum>QDateTimeEdit::YearSection</enum>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd   hh:mm:ss</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_50">
     <property name="text">
      <string>报警设置：</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="SetTime">
     <property name="text">
      <string>设置</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="ScreenCalibration">
     <property name="text">
      <string>屏幕校准</string>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QPushButton" name="UpdateProgram">
     <property name="text">
      <string>更新程序</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="WarningEnable">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>关闭</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>开启</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_21">
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>时间设置</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QPushButton" name="PlatformSelect">
     <property name="text">
      <string>平台选择</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>518</width>
       <height>108</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="1">
    <widget class="QComboBox" name="ScreenSave">
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="frame">
      <bool>false</bool>
     </property>
     <item>
      <property name="text">
       <string>关闭</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>5分钟</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>10分钟</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>30分钟</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>60 分钟</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>120分钟</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>导出文件</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
